metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -100,19 +103,59 @@ def test_s3_sensor_flow(pytestconfig) -> None:
     return
 
 
-# this test ensures the integration tests fail correctly
-def test_raise_failure_flow(pytestconfig) -> None:
+# This test ensures the integration tests fail correctly
+# and that when a test fails, an OpsGenie email is sent.
```

Comment:
nit: more precise would be to say that it ensures the Flow fails correctly, and when the flow fails, an OpsGenie email is sent.